src/style.css

```diff
@@ -0,0 +1,124 @@
+* {
+  box-sizing: border-box;
+  padding: 0;
+}
+
+body {
+  font-size: 16px;
+  font-family: "Avenir", sans-serif;
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-style: normal;
+  src: url("./fonts/Avenir-Book.ttf");
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-style: normal;
+  src: url("./fonts/Avenir-Heavy.ttf");
+  font-weight: 600;
+}
+
+.search-form--label:after {
+  position: absolute;
+  content: "";
+}
+
+.search-form--label--big:after {
+  top: 25px;
+  height: 19px;
+  width: 19px;
+  left: 26px;
+  position: absolute;
+  content: "";
+  background-size: contain;
+  background-repeat: no-repeat;
+  background-image: url("./images/Search.svg");
+}
+
+.search-form--label--small::after {
+  height: 11px;
+  width: 11px;
+  left: 13px;
+  top: 15px;
+  position: absolute;
+  content: "";
+  background-size: contain;
+  background-repeat: no-repeat;
+  background-image: url("./images/Search.svg");
+}
+
+.search-form--label {
+  width: 100%;
+  position: relative;
+}
+
+.search-form {
+  justify-content: center;
+  margin: 20px auto 0;
+  display: flex;
+}
+
+.search-form--input--small {
+  padding-left: 33px;
+  font-size: 14px;
+  height: 19px;
+}
+
+.search-form--input--big {
+  padding-left: 62px;
+  font-size: 16px;
+  line-height: 22px;
+  
+}
+
+.search-input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search-form--input {
+  height: 100%;
+  width: 100%;
+  font-family: inherit;
+  font-weight: 300;
+  outline: none;
+  color: #3d4e61;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}
+
+.search-form--input:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+  font-weight: 600;
+}
+
+.search-form--small {
+  width: 100%;
+  height: 42px;
+}
+
+.search-form--input--small::placeholder {
```

Comment:
**::placeholder** should not have bold font, only input text should be bold and have shadow